cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -539,14 +540,33 @@ private String pg(CQLTermNode node) throws QueryValidationException {
     if ("cql.allRecords".equalsIgnoreCase(node.getIndex())) {
       return "true";
     }
-    //determine if this table is real by checking in schema
-    //determine right here whether this node deals with a foreign term on right or left side
-    Table indexTable = checkForForeignLocationOfIndex(node);
-    if (indexTable != null) {
-      //we are doing a foreign key search
-      //determine foreign key linking tables
-      return subQuery(node.getIndex(), node, indexTable);
+
+//  //determine if this table is real by checking in schema
+//  //determine right here whether this node deals with a foreign term on right or left side
+//  Table indexTable = checkForForeignLocationOfIndex(node);
+//  if (indexTable != null) {
+//    //we are doing a foreign key search
+//    //determine foreign key linking tables
+//    return subQuery(node.getIndex(), node, indexTable);
+//  }
```

Comment:
Can you remove this commented-out code? We have git history if we want to restore.

Reply:
done